src/libs/Navigation/AppNavigator/AuthScreens.tsx

```diff
@@ -265,15 +250,7 @@ function AuthScreens({isUsingMemoryOnlyKeys, lastUpdateIDAppliedToClient, sessio
 
     return (
         <View style={styles.rootNavigatorContainerStyles(isSmallScreenWidth)}>
-            <RootStack.Navigator
-                isSmallScreenWidth={isSmallScreenWidth}
-                mode="modal"
-                // We are disabling the default keyboard handling here since the automatic behavior is to close a
-                // keyboard that's open when swiping to dismiss a modal. In those cases, pressing the back button on
-                // a header will briefly open and close the keyboard and crash Android.
-                // eslint-disable-next-line react/jsx-props-no-multi-spaces
-                keyboardHandlingEnabled={false}
```

Comment:
Why is this being removed?

Reply:
@allroundexperts Please, see [this discussion](https://github.com/Expensify/App/pull/29157#discussion_r1415122533) for the context